content/src/service/validations/ValidatorFactory.ts

```diff
@@ -11,7 +11,7 @@ export class ValidatorFactory {
       maxUploadSizePerTypeInMB: new Map([
         [EntityType.SCENE, 15],
         [EntityType.PROFILE, 15],
-        [EntityType.WEARABLE, 2]
+        [EntityType.WEARABLE, 3]
```

Comment:
Shouldn't we modify some configuration on the environment file too?

Reply:
This configuration is just used by the `ValidatorFactory` and there is no file for that